### ISSUE TITLE
API Authentication

### ISSUE THREAD

**Issue by adambirds** [ansible/awx#1947]
##### ISSUE TYPE
 - Bug Report
 - Documentation

##### COMPONENT NAME
 - API

##### SUMMARY
Running a **POST /api/v2/authtoken** returns `{"detail":"The requested resource could not be found."}`
Either the documentation is wrong at http://docs.ansible.com/ansible-tower/latest/html/towerapi/auth_token.html or their is something wrong with the api. 

##### ENVIRONMENT
* AWX version: 1.0.6.8
* AWX install method: docker on linux
* Ansible version:  2.5.2
* Operating System: CentOS 7
* Web Browser: Chromium

##### STEPS TO REPRODUCE

Either ran a **GET** or **POST** request to **/api/v2/authtoken** and it fails and come back with the following:

![screenshot from 2018-06-02 23-31-30](https://user-images.githubusercontent.com/13500254/40881440-b98861f2-66be-11e8-9684-68ce1329ad04.png)

##### EXPECTED RESULTS

It should return something similar to the below:

`{
    "token": "8f17825cf08a7efea124f2638f3896f6637f8745",
    "expires": "2013-09-05T21:46:35.729Z"
}`

##### ACTUAL RESULTS

`{"detail":"The requested resource could not be found."}`

##### ADDITIONAL INFORMATION

**Comment by ryanpetrello** [comment]
@adambirds the documentation you've referenced is for Ansible Tower 3.2.  In newer versions of awx, we've replaced the auth token implementation with OAuth2 tokens.  Take a peek at this documentation for an explanation on how to generate OAuth2 tokens:

https://github.com/ansible/awx/blob/devel/docs/auth/oauth.md

**Comment by rafis** [comment]
```
{"detail":"Authentication credentials were not provided. To establish a login session, visit /api/login/."}
```

**Comment by gespinoza22** [comment]
Any one get past this? Encountering `{"detail":"Authentication credentials were not provided. To establish a login session, visit /api/login/."}`

Even after following @ryanpetrello  method

**Comment by mark1973ryan** [comment]
Hi,

Don't know if you're still stuck on this but I've just started using the API.

The posts to /api/v2/applications/ and /api/v2/tokens/ using basic auth, obviously the user has to have rights on the box. I used our admin account. The second post gives you the tokens then you use those as bearer tokens in subsequent requests.

The ID for organisation can be picked from the URL when browsing them, here the application ID is 3.

https://awx.test.co.uk/#/applications/3?application_search=page_size:10;order_by:name
